### PR TITLE
Fixed itsdemongamez.is-a.dev

### PR DESCRIPTION
**Pull request by DemonGamez**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://shadowplayzyt22.github.io/Demon-Gamez-website.github.io/

### CODE REVIEW

**Comment by ShadowPlayzYT22** [comment]
This is the fixed version of the pull request, Hope i did everything correct.

File: domains/itsdemongamez.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+       "username": "ShadowPlayzYT22",
+       "email": "peterwyneg@gmail.com",
+       "discord": "1038747272701292584"
     },
+
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "Demon-Gamez-website.github.io"
```
Comment (DEV-DIBSTER) [inline domains/itsdemongamez.json:9]: 404 CNAME record. Please provide valid record.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/itsdemongamez.json (hunk 1 above)
Comment (ShadowPlayzYT22) [reply domains/itsdemongamez.json:9]: That is the correct cname, The cname should be the the github repostory name (100% sure), This cname worked before i changed the records to txt and stuff. Please correct me if im wrong. And by the way, I think you cannot reach websites using the cname.

Comment (nxrmqlly) [reply domains/itsdemongamez.json:9]: It isn't. Read the docs!

Comment (ShadowPlayzYT22) [reply domains/itsdemongamez.json:9]: hmmm, Okay

**Review by wdhdev — APPROVED** [review]

**Pull request merged**